test/new-e2e/tests/agent-platform/install-script/install_script_test.go

```diff
@@ -10,57 +10,40 @@ import (
 	filemanager "github.com/DataDog/datadog-agent/test/new-e2e/tests/agent-platform/common/file-manager"
 	helpers "github.com/DataDog/datadog-agent/test/new-e2e/tests/agent-platform/common/helper"
+	"github.com/DataDog/datadog-agent/test/new-e2e/tests/agent-platform/install"
 	"github.com/DataDog/datadog-agent/test/new-e2e/tests/agent-platform/install/installparams"
 	"github.com/DataDog/datadog-agent/test/new-e2e/tests/agent-platform/platforms"
-	"github.com/stretchr/testify/require"
 
-	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e"
-	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/client"
-	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/params"
-	"github.com/DataDog/datadog-agent/test/new-e2e/tests/agent-platform/install"
-	e2eOs "github.com/DataDog/test-infra-definitions/components/os"
-	"github.com/DataDog/test-infra-definitions/scenarios/aws/vm/ec2os"
-	"github.com/DataDog/test-infra-definitions/scenarios/aws/vm/ec2params"
+	osComp "github.com/DataDog/test-infra-definitions/components/os"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/ec2"
 
-	"testing"
+	"github.com/stretchr/testify/require"
 )
 
-var osVersion = flag.String("osversion", "", "os version to test")
-var platform = flag.String("platform", "", "platform to test")
-var cwsSupportedOsVersion = flag.String("cws-supported-osversion", "", "list of os where CWS is supported")
-var architecture = flag.String("arch", "x86_64", "architecture to test (x86_64, arm64))")
-var flavor = flag.String("flavor", "datadog-agent", "flavor to test (datadog-agent, datadog-iot-agent, datadog-dogstatsd, datadog-fips-proxy, datadog-heroku-agent)")
-var majorVersion = flag.String("major-version", "7", "major version to test (6, 7)")
+var (
+	osVersion             = flag.String("osversion", "", "os version to test")
+	platform              = flag.String("platform", "", "platform to test")
+	cwsSupportedOsVersion = flag.String("cws-supported-osversion", "", "list of os where CWS is supported")
+	architecture          = flag.String("arch", "x86_64", "architecture to test (x86_64, arm64))")
+	flavor                = flag.String("flavor", "datadog-agent", "flavor to test (datadog-agent, datadog-iot-agent, datadog-dogstatsd, datadog-fips-proxy, datadog-heroku-agent)")
+	majorVersion          = flag.String("major-version", "7", "major version to test (6, 7)")
+)
 
 type installScriptSuite struct {
-	e2e.Suite[e2e.VMEnv]
+	e2e.BaseSuite[environments.VM]
 	cwsSupported bool
 }
 
 func TestInstallScript(t *testing.T) {
-	osMapping := map[string]ec2os.Type{
-		"debian":      ec2os.DebianOS,
-		"ubuntu":      ec2os.UbuntuOS,
-		"centos":      ec2os.CentOS,
-		"amazonlinux": ec2os.AmazonLinuxOS,
-		"redhat":      ec2os.RedHatOS,
-		"rhel":        ec2os.RedHatOS,
-		"sles":        ec2os.SuseOS,
-		"windows":     ec2os.WindowsOS,
-		"fedora":      ec2os.FedoraOS,
-		"suse":        ec2os.SuseOS,
-		"rocky":       ec2os.RockyLinux,
-	}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Where did this go ?

cc @KevinFairise2

Reply:
We do this defined in test-infra already: https://github.com/DataDog/test-infra-definitions/blob/6295abafbaa844758b21a9e0e6a946aad1d16846/components/os/const.go